editoast/src/generated_data/error/mod.rs

```diff
@@ -87,6 +92,23 @@ impl<Ctx> GlobalErrorGenerator<Ctx> {
     }
 }
 
+/// An infra error with its hash
+struct ErrorWithHash {
+    error: InfraError,
+    /// Sha1 of the serialized error
+    hash: String,
+}
+
```

Comment:
Instead of doing a wrapper type of `InfraError` with the additional field, why not just implementing `Hash` directly on `InfraError`? It would avoid a new type and use the standard traits for that kind of usecase.

Reply:
We can't use the `std::Hash` trait with sha1. 
I don't want to use the `std::collections::hash_map::DefaultHasher` since there is no guarantee of the algorithm used and therefore of the hash produced.